src/test/resources/dockerRegistry/certs/domain.crt

```diff
@@ -0,0 +1,38 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Docker registry doesn't allow authentication unless it's over TLS: https://github.com/docker/distribution/blob/master/docs/deploying.md#native-basic-auth